projects/project-player-library/src/lib/pages/project-details/project-details.component.ts

```diff
@@ -15,11 +15,15 @@ ngOnChanges(changes: SimpleChanges): void {
   if (changes['projectDetails']) {
     this.learningResources = this.projectDetails?.learningResources || [];
     this.getCertificateCriteria();
+    this.getRecommendedFor();
```

Comment:
Why is this function called here?

Reply:
if the project details updates then it detects and recalculates

Reply:
So in that case why are we not doing it for getCategoryLabels()
Also this data is coming from api response right, we are not making any modifications to it from the frontend side also

Reply:
okay then i will remove it from the there.